DigitalRain/Raindrops/StandardRaindrop.cs

```diff
@@ -36,6 +37,7 @@ public StandardRaindrop(ColumnSpace space, double lifeSpan, Color symbolColor)
             LifeRemaining = lifeSpan;
 
             _symbolColor = symbolColor;
+            _startColor = Color.White;
```

Comment:
Eventually it'll be cool to play with different colors via the config

Reply:
That was the goal! But then I got distracted working on `global_config_access` branch